cypress/integration/fetch_items.js

```diff
@@ -42,6 +42,21 @@ describe("Show list of items", function() {
       cy.get('.purchasedItem');
       expect(cy.contains('Purchased Item'));
     });
+
+    it('Verifies that all four items categories are present on the list view', function() {
+      cy.visit('/list');
+      expect(cy.contains('Soon Items' && 'Kind of Soon Items' && 'Not Soon Items' && 'Inactive Items'));
+    });
```

Comment:
@gunderodd @michellejae thanks for adding tests to the codebase 💪 but this test is giving us false positives 😱 

Why? Well, because it is really testing that the list only contains `Inactive Items` and not every one of the items you list here! I made a video about this.

https://www.loom.com/share/b07ba5b67fff4963805091d83bd1c773

For now, what I'd suggest is to write a `expect(cy.contains(...))` for each one of the items.

🔮 in the future: You should avoid make your test dependent in _global state_ - what I mean by that is that you should take the app to a whole new state everytime you run a test. For this will be something like.

1. Get a Token
2. Create X amount of items
3. Verify that those items exist.

Steps 1 and 2 should repeat for every test. Right now, if someone goes and deletes the list you're using for testing this test will break.

Reply:
Thanks so much @alejandronanez , great points, I'm going to work on this tonight!

Reply:
@alejandronanez it appears to me ethan resolved this but I'm going to wait to click resolve until alejandro says it's good.